app/config.py

```diff
@@ -428,9 +426,7 @@ class Config(object):
     SESSION_COOKIE_SAMESITE = 'Lax'
 
     # Google Analytics
-    GOOGLE_ANALYTICS_ENABLED = str(True) == (os.getenv('GOOGLE_ANALYTICS_ENABLED', 'False'))
     GOOGLE_ANALYTICS_URL = os.getenv('GOOGLE_ANALYTICS_URL', 'https://www.google-analytics.com/collect')
-    GOOGLE_ANALYTICS_TID = os.getenv('GOOGLE_ANALYTICS_TID', 'UA-50123418-17')
```

Comment:
Nit: Could we also remove `GOOGLE_ANALYTICS_URL`? It looks like it's not used by code and only appears here.

Reply:
Yes, and I removed it.